src/main/java/uk/gov/companieshouse/api/accounts/validation/TangibleAssetsValidator.java

```diff
@@ -57,14 +44,25 @@ public TangibleAssetsValidator(CompanyService companyService, CurrentPeriodServi
     private static final String NET_BOOK_VALUE_CURRENT_PERIOD = ".net_book_value_at_end_of_current_period";
     private static final String NET_BOOK_VALUE_PREVIOUS_PERIOD = ".net_book_value_at_end_of_previous_period";
 
+    private CurrentPeriodService currentPeriodService;
```

Comment:
Always make any member vars that can be final. It makes it clear to anyone reading that the field does not change and proofs your code against misguided maintenance.